docs/deploy/security_groups.md

```diff
@@ -1,32 +1,68 @@
-# Security groups
+# Security Groups for Load Balancers
 
-The AWS Load Balancer Controller classifies security groups into two categories: frontend and backend.
+Use security groups to limit client connections to your load balancers, and restrict connections with nodes. The AWS Load Balancer Controller (LBC) defines two classifications of security groups: **frontend** and **backend**.
+
+- **Frontend Security Groups:** Determine the clients that can access the load balancers.
+- **Backend Security Groups:** Permit the load balancer to connect to targets, such as EC2 instances or ENIs.
 
 ## Frontend Security Groups
 
-Frontend security groups control which clients can access the load balancers. The frontend security groups can be configured with the `alb.ingress.kubernetes.io/security-groups` annotation on Ingress resources or `service.beta.kubernetes.io/aws-load-balancer-security-groups` annotation on Service resources. If the annotations are not specified, the LBC will create one security group per load balancer, allowing traffic from `inbound-cidrs` to `listen-ports`.
+Frontend security groups govern access to load balancers by specifying which clients can connect to them.
```

Comment:
Here or at the top of frontend_sg/index.md, can we talk about the use cases for frontend SGs:
- open from everywhere, running a public website 
- only open from another type of firewall (AWS WAF), running a public website with application protocol firewalling
- blocking by IP address/CIDRs, running a public website but blocking a country? though this is very inaccurate

- blocking by source SG IDs (recommended over IP/CIDR), running internal LBs

Reply:
Yes this is great feedback.